matrix.yml

```
@@ -2,7 +2,7 @@ x-cuda-prev-min: &cuda_prev_min { name: "cuda", version: "11.1" }
 x-cuda-prev-max: &cuda_prev_max { name: "cuda", version: "11.8" }
 x-cuda-curr-min: &cuda_curr_min { name: "cuda", version: "12.0" }
 x-cuda-curr-max: &cuda_curr_max { name: "cuda", version: "12.8" }
-x-cuda-curr-max: &cuda_curr_max_rapids { name: "cuda", version: "12.5" }
+x-cuda-curr-max-rapids: &cuda_curr_max_rapids { name: "cuda", version: "12.8" }
```

Discussed with @trxcllnt -- we will keep this at 12.5 for a moment so that we continue building 12.0, 12.5, and 12.8 containers.
```suggestion
x-cuda-curr-max-rapids: &cuda_curr_max_rapids { name: "cuda", version: "12.5" }
```